rxandroidble/src/main/java/com/polidea/rxandroidble2/RxBlePhy.java

```diff
@@ -0,0 +1,50 @@
+package com.polidea.rxandroidble2;
+
+import androidx.annotation.NonNull;
+
+import java.util.EnumSet;
+
+public enum RxBlePhy {
+    /**
+     * Unknown Bluetooth PHY. Used if the PHY couldn't be determined.
+     *
+     * @implNote If this value is used when calling the setPreferredPhy then it will default to PHY_1M
```

Comment:
It *should* be handled on the radio side, but I don't want to assume since there are different radios out there (I can only speak of nRF52840 radios). Since I don't want to assume, I changed to explicitly set it here:

https://github.com/ChandlerSystems/RxAndroidBle/blob/master/rxandroidble/src/main/java/com/polidea/rxandroidble2/internal/operations/PhyUpdateOperation.java#L50

Reply:
Completed in https://github.com/dariuszseweryn/RxAndroidBle/pull/840/commits/35215f8d7ae2b40172e348fa76928bd560d89538